src/components/dictionaryConcepts/components/helperFunction.js

```diff
@@ -31,3 +31,5 @@ export const classes = [
   'Test',
   'Procedure',
 ];
+
+export const internalMappingDefaultSource = 'CIEL';
```

Comment:
Don't we already have this constant somewhere in this code base?
Can you also google about naming conventions for constants?